src/assets/Standards/CollectibleStandards/ERC721/ERC721Standard.ts

```diff
@@ -1,50 +1,74 @@
-const ERC721_METADATA_INTERFACE_ID = '0x5b5e139f';
-const ERC721_ENUMERABLE_INTERFACE_ID = '0x780e9d63';
+import { abiERC721 } from '@metamask/metamask-eth-abis';
+import {
+  ERC721_INTERFACE_ID,
+  ERC721_METADATA_INTERFACE_ID,
+  ERC721_ENUMERABLE_INTERFACE_ID,
+  ERC721,
+} from '../../../../constants';
 
 export class ERC721Standard {
+  private web3: any;
+
+  constructor(web3?: any) {
+    this.web3 = web3;
```

Comment:
Similar comment as above.

Reply:
done here: https://github.com/MetaMask/controllers/pull/667/commits/8effcafddb6ca32d9566050ba50db569ceb00f07 and
here https://github.com/MetaMask/controllers/pull/667/commits/c9abc81e1eb69a9d1457fa7d2b0c08706bf292c4